UI/forms/OBSBasic.ui

```diff
@@ -961,7 +961,13 @@
    <attribute name="dockWidgetArea">
     <number>8</number>
    </attribute>
-   <widget class="QWidget" name="dockWidgetContents_7">
```

Comment:
This should be a QFrame within a QWidget similar to the Scenes and Sources dock

![image](https://user-images.githubusercontent.com/1554753/129494549-0a577eea-73c3-4a54-b369-4a29cb6084a9.png)

Reply:
This is out of scope for this PR so I'm retracting it